docs/_tutorials/index.md

```diff
@@ -3,7 +3,7 @@ title: Tutorials
 ---
 **Note:** Make sure you've [installed Wash]({{ '/getting_started' | relative_url }}) before going any further.
 
-This series of hands-on tutorials introduces you to Wash. Each tutorial assumes that you have the [test environment set-up](00_test_environment), and that you have an understanding of Wash’s key abstractions. If you're unfamiliar with Wash, start with the [Fundamentals](01_fundamentals/plugins-actions-entries). You can complete the tutorials in any order. If you're not sure where to start, here are a few good options: 
+This series of (optional) hands-on tutorials introduces you to Wash. Each tutorial assumes that you have the [test environment set-up](00_test_environment), and that you have an understanding of Wash’s key abstractions. If you're unfamiliar with Wash, start with the [Fundamentals](01_fundamentals/plugins-actions-entries). You can complete the tutorials in any order. If you're not sure where to start, here are a few good options: 
```

Comment:
I still really do want to remove "Fundamentals" b/c I think the changes here better (and more concisely) captures all of that material.

Reply:
Still need to update some prompts with the new "Welcome" message so want to make sure we figure that out first.

Reply:
Also, much of "Fundamentals" is the type of stuff that you'll immediately "get" upon using Wash. Maybe something about Attributes/Metadata could be useful, but outside of that I don't think it has much value.

Reply:
@MikaelSmith Thoughts on this? I think at the very least "Fundamentals" => "Attributes/Metadata".

Reply:
Changing to Attributes/Metadata and slimming it down (if some parts seem redundant) seem reasonable.

Reply:
Done.